pytket/extensions/cutensornet/mps/mps.py

```diff
@@ -176,16 +171,75 @@ def __init__(
             raise TypeError(
                 f"Value of float_precision must be in {allowed_precisions}."
             )
+        self.zero = value_of_zero
 
-        self._lib = libhandle
-        self._logger = set_logger("MPS", level=loglevel)
+        if value_of_zero > self._atol / 1000:
+            warnings.warn(
+                "Your chosen value_of_zero is relatively large. "
+                "Faithfulness of final fidelity estimate is not guaranteed.",
+                UserWarning,
+            )
 
-        #######################################
-        # Initialise the MPS with a |0> state #
-        #######################################
+        self.k = k
+        self.optim_delta = 1e-5
+        self.loglevel = loglevel
+
+    def copy(self) -> ConfigMPS:
+        """Standard copy of the contents."""
+        return ConfigMPS(
+            chi=self.chi,
+            truncation_fidelity=self.truncation_fidelity,
+            k=self.k,
+            optim_delta=self.optim_delta,
+            float_precision=self._real_t,  # type: ignore

```

Comment:
I have noticed that this is missing value_of_zero and loglevel.

Reply:
Are you planning to add those?

Reply:
Yes, I now see they should be added.

Reply:
Done.